livingatlas/pipelines/src/main/java/au/org/ala/pipelines/transforms/TransformsFactory.java

```diff
@@ -0,0 +1,242 @@
+import org.gbif.kvs.KeyValueStore;
+import org.gbif.kvs.geocode.LatLng;
+import org.gbif.kvs.grscicoll.GrscicollLookupRequest;
+import org.gbif.kvs.species.SpeciesMatchRequest;
+import org.gbif.pipelines.common.PipelinesVariables.Pipeline.Interpretation.RecordType;
+import org.gbif.pipelines.common.beam.options.InterpretationPipelineOptions;
+import org.gbif.pipelines.core.config.model.PipelinesConfig;
+import org.gbif.pipelines.core.functions.SerializableSupplier;
+import org.gbif.pipelines.core.pojo.HdfsConfigs;
+import org.gbif.pipelines.core.utils.FsUtils;
+import org.gbif.pipelines.core.ws.metadata.MetadataServiceClient;
+import org.gbif.pipelines.factory.ClusteringServiceFactory;
+import org.gbif.pipelines.factory.FileVocabularyFactory;
+import org.gbif.pipelines.factory.GeocodeKvStoreFactory;
+import org.gbif.pipelines.factory.GrscicollLookupKvStoreFactory;
+import org.gbif.pipelines.factory.KeygenServiceFactory;
+import org.gbif.pipelines.factory.MetadataServiceClientFactory;
+import org.gbif.pipelines.factory.NameUsageMatchStoreFactory;
+import org.gbif.pipelines.factory.OccurrenceStatusKvStoreFactory;
+import org.gbif.pipelines.io.avro.ExtendedRecord;
+import org.gbif.pipelines.keygen.HBaseLockingKey;
+import org.gbif.pipelines.transforms.common.ExtensionFilterTransform;
+import org.gbif.pipelines.transforms.common.FilterRecordsTransform;
+import org.gbif.pipelines.transforms.common.UniqueGbifIdTransform;
+import org.gbif.pipelines.transforms.common.UniqueIdTransform;
+import org.gbif.pipelines.transforms.converters.OccurrenceExtensionTransform;
+import org.gbif.pipelines.transforms.core.BasicTransform;
+import org.gbif.pipelines.transforms.core.EventCoreTransform;
+import org.gbif.pipelines.transforms.core.GrscicollTransform;
+import org.gbif.pipelines.transforms.core.LocationTransform;
+import org.gbif.pipelines.transforms.core.TaxonomyTransform;
+import org.gbif.pipelines.transforms.core.TemporalTransform;
+import org.gbif.pipelines.transforms.core.VerbatimTransform;
+import org.gbif.pipelines.transforms.extension.AudubonTransform;
+import org.gbif.pipelines.transforms.extension.ImageTransform;
+import org.gbif.pipelines.transforms.extension.MeasurementOrFactTransform;
+import org.gbif.pipelines.transforms.extension.MultimediaTransform;
+import org.gbif.pipelines.transforms.metadata.DefaultValuesTransform;
+import org.gbif.pipelines.transforms.metadata.MetadataTransform;
+import org.gbif.pipelines.transforms.specific.ClusteringTransform;
+import org.gbif.pipelines.transforms.specific.GbifIdAbsentTransform;
+import org.gbif.pipelines.transforms.specific.GbifIdTransform;
+import org.gbif.pipelines.transforms.specific.GbifIdTransform.GbifIdTransformBuilder;
+import org.gbif.pipelines.transforms.specific.IdentifierTransform;
+import org.gbif.rest.client.geocode.GeocodeResponse;
+import org.gbif.rest.client.grscicoll.GrscicollLookupResponse;
+import org.gbif.rest.client.species.NameUsageMatch;
+
+@Getter
```

Comment:
Needs a class comment
And, probably comments for each method so that innocent programmers know what's what.

Reply:
comments added